src/constructs/aws/Queue.ts

```diff
@@ -40,6 +40,11 @@ const QUEUE_DEFINITION = {
             minimum: 1,
             maximum: 10,
         },
+        maxBatchingWindow: {
```

Comment:
I'm not sure of the name, each time I have to search for it because it's not clear what it does just by reading the name.

We often try to think outside of the cloudformation/cdk names to find something meaningful if possible.

I unfortunately have no idea on a better name in this case 😅

Reply:
maxBatchingWindowInSeconds?
batchDelay?

Reply:
@t-richard I'll update the docs -- Just wasn't sure if you would be ok with the direction in terms of changing how the visibility timeout is calculated :)

Reply:
> I unfortunately have no idea on a better name in this case 😅

Same, I don't see a better alternative. `maxBatchingWindowInSeconds` is good but very long 🤷

Reply:
`batchWaitPeriod` ? Not sure about this one too but it keeps the discussion going :smile:

Reply:
batchBuffer

Reply:
Since we don't see an obvious alternative name, let's stick with `maxBatchingWindow` then.

Thanks for bringing this up though, this is an interesting exercise. It just doesn't always pay off  😅

Reply:
in aws docs the name for this prop is 'MaximumBatchingWindowInSeconds'